examples/vmod_blobs/README.md

```diff
@@ -0,0 +1,51 @@
+# VMOD Blobs
+
+This example demonstrates how to use VCL_BLOB arguments in Varnish modules written in Rust.
+
+## Overview
+
+Starting with this version of varnish-rs, you can accept BLOB data as input arguments to your VMOD functions. In Rust, BLOBs are represented as `&[u8]` slices, providing safe and efficient access to binary data.
+
+## Features Demonstrated
+
+This example VMOD provides several functions that work with BLOB data:
+
+- `blob_length(data: &[u8]) -> i64` - Returns the length of a BLOB
```

Comment:
is blob data always readonly?

Reply:
it's not (it's C underneath!), but here it's implemented as read-only

Reply:
do you want to mention that it's readonly for now in the docs somewhere?

Reply:
actually, we have this line in `convert.rs`:

```//! | `&[u8]` | <- | `VCL_BLOB` |```

that should be enough?

Reply:
hum, except that block doesn't show in the docs, I'll do that in an other PR